Makefile

```diff
@@ -0,0 +1,9 @@
+FILE=./VERSION
+RESUMIC_VERSION = `cat $(FILE)`
+GITCOMMIT = `git describe --tags --always --dirty`
+CODEPATH=`go list -m`
+build: ./cmd/version.go
+	go get
```

Comment:
Gotcha. Is `packr2 build` supposed to run instead of `go build`? And should I get rid of the `go get` at the top of the `build` rule?

Reply:
Personally, I prefer to use `packr2` followed by `go build` instead of `packr2 build`. I think using the default `go build` make it a little more readable.
As `go build` will run the go get before the compile having a separated `go get` is not needed.

Reply:
Makefile now runs just `packr2`, then `go build`, then `packr2 clean`

Reply:
How is `packr2` included in the build environment? Travis seems to be failing because it can't find `packr2`: https://travis-ci.com/jahzielv/schema/jobs/192206152

Reply:
packr2 binary could be installed with the:
`go get github.com/gobuffalo/packr/v2/packr2`

Reply:
Added to latest commit!